cms/static/cms/js/require-config.js

```diff
@@ -52,7 +52,7 @@
             "text": 'js/vendor/requirejs/text',
             "underscore": "common/js/vendor/underscore",
             "underscore.string": "common/js/vendor/underscore.string",
-            "backbone": "js/vendor/backbone-min",
+            "backbone": "common/js/vendor/backbone-min",
```

Comment:
Sorry I didn't get to the review in time. Nit: we are including the non-minified versions of libraries in development, and then relying upon the pipeline's optimization to minify them. You can see ``underscore`` and ``underscore.string`` above are not minified.